client/components/AssignDoi/AssignDoi.tsx

```diff
@@ -1,23 +1,25 @@
-import React, { useCallback, useReducer, useEffect, useMemo, useRef } from 'react';
```

Comment:
Much of the code in this file was refactored to use the extracted `ReviewDepositCallout` and `SubmitDepositButton` components.